datafusion/optimizer/Cargo.toml

```diff
@@ -33,16 +33,19 @@ name = "datafusion_optimizer"
 path = "src/lib.rs"
 
 [features]
-default = ["unicode_expressions"]
-unicode_expressions = []
+default = ["unicode_expressions", "crypto_expressions", "regex_expressions"]
+crypto_expressions = ["datafusion-physical-expr/crypto_expressions"]
+regex_expressions = ["datafusion-physical-expr/regex_expressions"]
+unicode_expressions = ["datafusion-physical-expr/unicode_expressions"]
+dictionary_expressions = ["datafusion-physical-expr/dictionary_expressions"]
```

Comment:
Otherwise, datafusion-optimizer always pulls in these features from datafusion-physical-expr.

Reply:
For example, we cannot exclude `crypto_expressions` from it.